TypeInjections/Tool/Tool.fs

```diff
@@ -7,91 +7,102 @@ open LibGit2Sharp
 open System.IO
 open System
 open TypeInjections
+open CommandLine
 
 module Tool =
 
+
+    [<Verb("local", HelpText = "typecart local [old project path] [new project path] [outputPath]")>]
+    type LocalOptions =
+        { [<Option('o', "old", Required = false, HelpText = "folder path of old project")>]
+          oldFolder: string
+          [<Option('n', "new", Required = false, HelpText = "folder path of new project")>]
+          newFolder: string
+          [<Option('p', "output", Required = false, HelpText = "path of folder to place typeCart files")>]
+          outputFolder: string
+          [<Option('f', "file", Required = false, HelpText = "specify filename(s) you want to run typeCart on")>]
+          file: seq<string> }
+
+    [<Verb("git", HelpText = "typecart git [old commit] [new commit] [outputPath]")>]
+    type GitOptions =
+        { [<Option('o', "old", Required = true, HelpText = "commit hash of old version")>]
+          oldHash: string
+          [<Option('n', "new", Required = true, HelpText = "commit hash of new version")>]
+          newHash: string
+          [<Option('p', "output", Required = true, HelpText = "path of folder to place typeCart files")>]
+          outputFolder: string
+          [<Option('e', "entrypoint", Required = false, HelpText = "specify the filename(s) you want to run typeCart on")>]
+          file: seq<string>
+          [<Option('c', "clone", Required = false, HelpText = "input a git URL")>]
+          url: string
+          [<Option('f', "force", Required = false, HelpText = "overwrite the Old and New folders in the Output folder")>]
+          force: bool }
+
     //NOTE 'location' refers to either the github url OR path to repo on local computer
     let checkoutCommit (hash: string) (location: string) (outputPath: string) : unit =
-        Utils.log $"path to repo is {outputPath}"
-
         Repository.Clone(location, outputPath) |> ignore
         // get Repository object of whatever was cloned above
         let newRepo = new Repository(outputPath)
         // checkout repo at given commit
         let commitRepo = newRepo.Lookup<Commit>(hash)
```

Comment:
This would fail if the current directory isn't from git, we should handle that failure

Reply:
Handled, and useful error message is given